lib/src/translator.dart

```diff
@@ -77,8 +77,10 @@ class Translator {
 
     final namespaces = <String>[
       namespace,
-      if (options.fallbackNamespace != null) options.fallbackNamespace!,
     ];
+    if(options.fallbackNamespaces?.isNotEmpty ?? false){
+      namespaces.addAll(options.fallbackNamespaces!);
+    }
```

Comment:
Formatting here is missing some whitespaces.

Reply:
Rather than adding it to the list, could you keep it inside the list declaration?

```dart
final namespaces = <String>[
  namespace,
  if (options.fallbackNamespaces != null) ...options.fallbackNamespaces!,
];
```

Reply:
Will do, forgot about the spread operator for a moment